spec/requests/items_spec.rb

```diff
@@ -0,0 +1,118 @@
+require 'rails_helper'
+
+RSpec.describe "Items API", type: :request do
+	let!(:todo) {create(:todo)}
+	let!(:items) {create_list(:item, 20, todo_id: todo.id)}
+	let(:todo_id) {todo.id}
+	let(:id) {items.first.id}
+
+	#GET /todos/:todo_id/items
+	describe "GET /todos/:todo_id/items" do
+		before {get "/todos/#{todo_id}/items"}
+
+		context "when valid todo items record exists." do
+			it "returns todo items" do
+				expect(parsed_response.size).to eq(20)
+			end
```

Comment:
PLease also test parsed_response value. Be sure to match the parsed_response with the todos record created in test db.